### PR TITLE
Add one_hot_v2 operator support

### PR DESCRIPTION
**Pull request by yeliang2258**
1. Add one_hot_v2 operator support

### CODE REVIEW

File: paddle2onnx/mapper/tensor/one_hot_v2.cc [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+// limitations under the License.
+
+#include "paddle2onnx/mapper/tensor/one_hot_v2.h"
+
+namespace paddle2onnx {
+REGISTER_MAPPER(one_hot_v2, OneHotV2Mapper)
+
+int32_t OneHotV2Mapper::GetMinOpset(bool verbose) {
+  if (allow_out_of_range_) {
+    Error() << "allow_out_of_range can not be true in one_hot_v2." << std::endl;
```
Comment (jiangjiajun) [inline paddle2onnx/mapper/tensor/one_hot_v2.cc]: 是cannot be true 还是is not supported

File: paddle2onnx/mapper/tensor/one_hot_v2.cc [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+    return -1;
+  }
+  auto output_info = parser_->GetOpOutput(block_idx_, op_idx_, "Out");
+  if (output_info[0].dtype != dtype_) {
+    Error() << "dtype attribute and output dtype do not match." << std::endl;
+    return -1;
+  }
+  return 9;
```
Comment (jiangjiajun) [inline paddle2onnx/mapper/tensor/one_hot_v2.cc:32]: Logger(verbose, 9) << RequireOpset(9) << std::endl;

File: paddle2onnx/mapper/tensor/one_hot_v2.cc [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+}
+
+void OneHotV2Mapper::Opset9() {
+  auto input_info = parser_->GetOpInput(block_idx_, op_idx_, "X");
+  auto output_info = parser_->GetOpOutput(block_idx_, op_idx_, "Out");
```
Comment (jiangjiajun) [inline paddle2onnx/mapper/tensor/one_hot_v2.cc]: 同GatherND

**Review by jiangjiajun — CHANGES_REQUESTED** [review]

File: paddle2onnx/mapper/tensor/one_hot_v2.cc (hunk 1 above)
Comment (yeliang2258) [reply paddle2onnx/mapper/tensor/one_hot_v2.cc]: is not supported

**Review by jiangjiajun — APPROVED** [review]

**Pull request merged**